src/reducers/profile-view.js

```diff
@@ -922,10 +923,17 @@ export const selectorsForThread = (
       getProcessedMarkersThread,
       ProfileData.getTracingMarkers
     );
+
+    const getNetworkMarkers = createSelector(
+      getProcessedMarkersThread,
```

Comment:
Yeah, I believe the marker chart search PR #1216 changes this slightly to used the search filtered markers.